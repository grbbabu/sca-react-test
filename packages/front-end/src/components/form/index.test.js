import React from 'react';
import renderer from 'react-test-renderer';
import Form from './index';

it('renders correctly when there are no input to the form', () => {
    const tree = renderer
        .create(
            <Form>
                <label htmlFor="first">
                    First Name
                    <input type="text" id="first" name="first" />
                </label>
                <button type="submit">Submit</button>
            </Form>
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});
