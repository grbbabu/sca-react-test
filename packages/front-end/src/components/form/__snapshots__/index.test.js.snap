// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no input to the form 1`] = `
<form
  className={null}
>
  <label
    htmlFor="first"
  >
    First Name
    <input
      id="first"
      name="first"
      type="text"
    />
  </label>
  <button
    type="submit"
  >
    Submit
  </button>
</form>
`;
